sdk/webpubsub/Azure.Messaging.WebPubSub.Client.Protobuf/src/Azure.Messaging.WebPubSub.Client.Protobuf.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>1.0.0-beta.1</Version>
+    <PackageId>Azure.Messaging.WebPubSub.Client.Protobuf</PackageId>
+    <Description>Client Protobuf Protocol for Azure WebPubSub</Description>
+    <PackageTags>azure;webpubsub.client</PackageTags>
+    <NoWarn>$(NoWarn);0067</NoWarn>
+    <IncludeOperationsSharedSource>true</IncludeOperationsSharedSource>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Remove="Protos\webpubsub.client.proto" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Azure.Core" />
+    <PackageReference Include="System.Threading.Channels" />
+    <PackageReference Include="Azure.Messaging.WebPubSub.Client" VersionOverride="1.0.0-beta.2" />
+    <PackageReference Include="Google.Protobuf" VersionOverride="3.21.9" />
```

Comment:
I don't think we should do it without a discussion. And for the discussion, it would be good to see concrete numbers justifying this additional dependency.

Reply:
@JeffreyRichter, @johanste, was this reviewed and approved by the stewardship?

Reply:
We've never reviewed anything trying to use proctocol buffers; we would not allow it.

Reply:
We never reviewed any service trying to use protocol buffers; it is not allowed for Azure services.

Reply:
It's not grpc scenario. And I've written an email for further discussion.